django_project/realtime/tasks/realtime/test/test_realtime_tasks.py

```diff
@@ -1,26 +1,42 @@
 # coding=utf-8
+import logging
+import os
+import shutil
+import time
+import unittest
 from datetime import datetime
 
-import os
 import pytz
+from celery.result import AsyncResult
 from django import test
 from django.apps import apps
 from django.core.files.base import File
 
+from realtime.app_settings import EARTHQUAKE_MONITORED_DIRECTORY, LOGGER_NAME
 from realtime.models.ash import Ash
+from realtime.models.earthquake import Earthquake
 from realtime.models.volcano import Volcano
+from realtime.tasks import check_processing_task
+from realtime.tasks.realtime.celery_app import app as realtime_app
 
 __copyright__ = "Copyright 2016, The InaSAFE Project"
 __license__ = "GPL version 3"
 __email__ = "info@inasafe.org"
 __revision__ = ':%H$'
 
 
-class TestAshTasks(test.SimpleTestCase):
+LOGGER = logging.getLogger(LOGGER_NAME)
+
+
+@unittest.skipUnless(
+    realtime_app.control.ping(),
```

Comment:
I found a nice way to check that the worker is connected :D

Reply:
Nice.